openfisca/requirements.txt

```diff
@@ -1,5 +1,7 @@
+
 gunicorn>=19.1.1
-openfisca_web_api==3.0.0
-openfisca_france==10.2.0
-git+https://github.com/sgmap/openfisca-paris.git@a4b6f324f48e465b89ffd01d25cea8ff68202718#egg=openfisca-paris
-git+https://github.com/sgmap/openfisca-rennesmetropole.git@3487bb332f4373c3123ebdd7e31ecd4da5d82548#egg=openfisca-RennesMetropole
+openfisca_web_api[paster]==3.4.0
```

Comment:
Why do we install the `paster` optional dependencies when we also install `gunicorn` above?